server/datastore/mysql/cron_stats.go

```diff
@@ -15,7 +17,7 @@ func (ds *Datastore) GetLatestCronStats(ctx context.Context, name string) ([]fle
 	stmt := `
 (
 	SELECT
-		id, name, instance, stats_type, status, created_at, updated_at
+		id, name, instance, stats_type, status, coalesce(errors, '') as errors, created_at, updated_at
```

Comment:
Why does this method need to return errors? Are they used somewhere?

This method gets called every time we trigger a job, so it should be somewhat efficient.

Reply:
Presently it's only used in [the integration test](https://github.com/fleetdm/fleet/blob/cbfc575e9b1f86e57ab38c4a85b1ce6e44bc95ba/server/datastore/mysql/cron_stats_test.go#L22-L64) to ensure that errors make it all the way to the database.  I don't think returning this extra column (which will usually be NULL) with 2 rows will have a real performance impact, but I could take it out and have that test do a direct SQL query instead if you prefer.  Feels like something called `GetLatestCronStats` should return this info, but I'm ambivalent about returning it now vs. waiting until other code needs to access it.

Reply:
Yes, lets take it out if it is just for testing. We don't want to be debugging an issue with a failing cron, and then this query potentially making things worse.

Reply:
Done!